scripts/fast-reboot

```diff
@@ -41,6 +41,18 @@ docker kill lldp > /dev/null
 # Kill teamd, otherwise it gets down all LAGs
 docker kill teamd > /dev/null
 
+# Kill swss before stopping syncd
+docker kill swss > /dev/null
+
+# Gracefully stop syncd
+docker exec -ti syncd /usr/bin/syncd_request_shutdown --cold > /dev/null
```

Comment:
will this break the mellanox platform?

Reply:
I see. I missed the other change. Sorry.

Regards,
Ying